src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/search/SortField.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.search;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum SortField {
+
+    DUE_DATE("dueDate"),
+    TASK_TITLE("taskTitle"),
+    LOCATION_NAME("locationName"),
```

Comment:
Looks like we no longer need to sort by **locationName**. Its crossed out on the ticket